src/withWebViewBridge.tsx

```diff
@@ -18,6 +19,19 @@ export type BaseWebViewPropsType = {
     onMessage?: (event: any) => any;
 }
 
+const getCircularReplacer = () => {
```

Comment:
should we be citing our sources? (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cyclic_object_value#Examples)